packages/core/admin/admin/src/content-manager/components/RelationInputDataManager/RelationInputDataManager.js

```diff
@@ -153,7 +169,7 @@ export const RelationInputDataManager = ({
       label={`${formatMessage({
         id: intlLabel.id,
         defaultMessage: intlLabel.defaultMessage,
-      })} ${initialData[name]?.count !== undefined ? `(${initialData[name].count})` : ''}`}
+      })} ${totalRelations > 0 ? `(${totalRelations})` : ''}`}
```

Comment:
Same here: I think tests would be very beneficial now that this has become more complex.

Also: do you have an idea how we could move this into the translation string? The way we do this still bothers me a bit 🤦🏼

Reply:
I don't think we can because the label comes from the API 🤔